### ISSUE TITLE
Include ChromeDriver

### ISSUE THREAD

**Issue by DavertMik** [sveneisenschmidt/selenium-server-standalone#26]
Currently Selenium is usually not enough to launch tests as it may require ChromeDriver or GeckoDriver to work on. Would it be possible for this project to include ChromeDriver as well?

**Comment by asgrim** [comment]
I don't think this is within the scope of this project; it doesn't make sense to bundle any browser-specific driver within. I understand from a convenience point of view, but it would make understanding the release schedules pretty hectic. Which driver(s) would we bundle? Why? How often are they updated (and, by proxy, how often would we have to release). Seems like an unnecessary maintenance burden to me...

**Comment by DavertMik** [comment]
I was just looking to something similar to https://www.npmjs.com/package/selenium-standalone and I didn't find any good library in PHP except this one :+1: 
It would be cool if there was a way for newbies to launch selenium without pain by installing just a single package. Something similar to what Laravel Dusk does. 

---

As for maintenance I think it can be done with a simple PHP script: 

* check for updates
* download new binaries
* run basic tests
* make PR

**Comment by asgrim** [comment]
Something to download the latest driver could be workable, yeah. But I wouldn't bundle the binaries in this repo (as the JAR already is). :+1:

**Comment by DavertMik** [comment]
Possible solution:

1. install `laravel/dusk`, 
2. copy chromedriver out of it
3. remove laravel dusk
4. PROFIT 
 :rofl:

**Comment by sveneisenschmidt** [comment]
There could be additional repositories/packages setup to handle this. selenium-driver-chrome, selenium-driver-firefox, ....

**Comment by DavertMik** [comment]
Btw there is already an initiative from Joomla you know about. https://github.com/joomla-projects/selenium-server-standalone

Did you have a chance to get more steps into merging the forces?

**Comment by sveneisenschmidt** [comment]
@DavertMik I am open to further pursue this. https://github.com/joomla-projects/selenium-server-standalone/issues/25
